### PR TITLE
Bug_fix_01

### PR DESCRIPTION
**Pull request by ue71603**
First part of bugs / typos found during reading for acceptance.

### CODE REVIEW

File: xsd/netex_framework/netex_genericFramework/netex_contact_support.xsd [hunk 1]
```diff
@@ -97,9 +97,9 @@ Rail transport, Roads and Road transport
 		<xsd:restriction base="xsd:NMTOKEN">
 			<xsd:enumeration value="any"/>
 			<xsd:enumeration value="information"/>
-			<xsd:enumeration value="reservations"/>
+			<xsd:enumeration value="reservation"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_genericFramework/netex_contact_support.xsd:100]: Where do the plurals come from?

File: xsd/netex_framework/netex_utility/netex_units.xsd [hunk 2]
```diff
@@ -125,7 +125,7 @@ Speed = Metres per second.</xsd:documentation>
 	</xsd:simpleType>
 	<xsd:simpleType name="VoltageType">
 		<xsd:annotation>
-			<xsd:documentation>Voltage time in SI Volts.ed by http://www.ordnancesurvey.co.uk/xml/resource/units.xml#metres.</xsd:documentation>
+			<xsd:documentation>Voltage type in SI Volts.ed by http://www.ordnancesurvey.co.uk/xml/resource/units.xml#metres.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_utility/netex_units.xsd:128]: ed by?

**Comment by skinkie** [comment]
Reasons for the example file deletions?

File: xsd/netex_framework/netex_genericFramework/netex_contact_support.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/netex_framework/netex_genericFramework/netex_contact_support.xsd:100]: They were in the standard. But all to other enum values are not plurals, so i harmonized them.

File: xsd/netex_framework/netex_utility/netex_units.xsd (hunk 2 above)
Comment (ue71603) [reply xsd/netex_framework/netex_utility/netex_units.xsd]: changed. please review @skinkie  @Aurige

**Review by skinkie — APPROVED** [review]

**Pull request merged**